autoconfigure/collector-sqs/src/test/java/zipkin/autoconfigure/collector/sqs/ZipkinSQSCollectorAutoConfigurationTest.java

```diff
@@ -80,18 +84,36 @@ public void provideCollectorComponent_setsZipkinSqsCollectorProperties() {
     addEnvironment(context, "zipkin.collector.sqs.queue-url:" + sqsRule.queueUrl());
     addEnvironment(context, "zipkin.collector.sqs.wait-time-seconds:1");
     addEnvironment(context, "zipkin.collector.sqs.parallelism:3");
+    addEnvironment(context, "zipkin.collector.sqs.aws-access-key-id: x");
+    addEnvironment(context, "zipkin.collector.sqs.aws-secret-access-key: x");
     context.register(PropertyPlaceholderAutoConfiguration.class,
-        ZipkinSQSCollectorAutoConfiguration.class, InMemoryConfiguration.class);
+        ZipkinSQSCollectorAutoConfiguration.class, ZipkinSQSCredentialsAutoConfiguration.class, InMemoryConfiguration.class);
     context.refresh();
 
     ZipkinSQSCollectorProperties properties = context.getBean(ZipkinSQSCollectorProperties.class);
 
-
     assertThat(properties.getQueueUrl()).isEqualTo(sqsRule.queueUrl());
     assertThat(properties.getWaitTimeSeconds()).isEqualTo(1);
     assertThat(properties.getParallelism()).isEqualTo(3);
   }
 
+  @Test
+  public void provideSecurityTokenService_whenAwsStsRoleArnIsSet() {
+    context = new AnnotationConfigApplicationContext();
+    addEnvironment(context, "zipkin.collector.sqs.queue-url:" + sqsRule.queueUrl());
+    addEnvironment(context, "zipkin.collector.sqs.wait-time-seconds:1");
+    addEnvironment(context, "zipkin.collector.sqs.aws-access-key-id: x");
```

Comment:
These are actually the default values for ElasticMQ provided by SQSRule.  In this case its only used to provide a valid Queue URL otherwise the SQS Client errors out.